CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
  * Global Registration: As a user, I can easily setup certs in the /etc/containers/certs.d/hostname/ directory ([#38455](https://projects.theforeman.org/issues/38455), [6e1f42d9](https://github.com/Katello/katello.git/commit/6e1f42d98966b41bf5b55957d24769976da9f3cf))
 
 ### Hosts
- * As a user I want to be able to change the host collection of multiple hosts ([#38829](https://projects.theforeman.org/issues/38829), [6625fe75](https://github.com/Katello/katello.git/commit/6625fe75ffea36a14666d26d1f8d314851b970a0))
+ * As a user I want to be able to change the host collection of multiple hosts ([#38829](https://projects.theforeman.org/issues/38829), [6625fe75](https://github.com/Katello/katello.git/commit/6625fe75ffea36a14666d26d1f8d314851b970a0), [7978ce3c](https://github.com/Katello/katello.git/commit/7978ce3c117d683fb91271968afa97c913ce22ad))
```

Comment:
The new commit here did not make it into Katello 4.19.0. @lfu @jeremylenz should it have?